.github/workflows/ci.yml

```diff
@@ -19,6 +19,9 @@ concurrency:
 permissions:
   contents: read
 
+env:
```

Comment:
Why we need to ass this?

Reply:
I removed it again, I tried to fix the CI but as the environment is not inherited by called workflows it doesn't work and the current CI is effectively broken/stuck until forcefully merging a PR with failing CI.